packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -5369,6 +5369,48 @@ class Flow extends MultiChildRenderObjectWidget {
   }
 }
 
+class _SemanticsTagWidget extends SingleChildRenderObjectWidget {
```

Comment:
Can we just roll this functionality into the Semantics widget? That is, the Semantics widget should have a `tag` property to take a `SemanticsTag`.

Reply:
what is the correct behavior on the setting semantics tags in SemanticsNode? does it add the tag to itself or does it add the tags for the subtree only?

Reply:
What do you mean? We only have the `addTagForChildren` API, no?

Reply:
oh, yes, so the behavior should be adding tag for the semantics subtree except itself if it also introduce a semantics node?

Reply:
Yeah